src/module.ts

```diff
@@ -586,6 +577,9 @@ for(let i=0; i<LAYER_LIMIT; i++){
     if(options[option].hasOwnProperty("description")){
       options[option].description = options[option].description.replace("${i+1}", (i+1).toString());
     }
+    if(options[option].hasOwnProperty("category")){
+      options[option].category = options[option].category.replace("${i+1}", (i+1).toString());
+    }
```

Comment:
These 3 iterations feel like they could be merged into a single function.

for var thing in ["description", "category", "defaultVault"]
   var val = options[option][thing];
   if val == undefined || not typeof(val) === 'string'
       continue
   
    options[option][thing] = val.replace("${i+1}", (i+1).toString())